inference-engine/src/mkldnn_plugin/nodes/mkldnn_normalize_node.cpp

```diff
@@ -648,8 +648,11 @@ MKLDNNNormalizeL2Node::MKLDNNNormalizeL2Node(const std::shared_ptr<ngraph::Node>
     std::string errorMessage;
     if (isSupportedOperation(op, errorMessage)) {
         errorPrefix = "NormalizeL2 node with name '" + getName() + "' ";
-        eps = std::dynamic_pointer_cast<const ngraph::op::v0::NormalizeL2>(op)->get_eps();
+        const auto norm = std::dynamic_pointer_cast<const ngraph::op::v0::NormalizeL2>(op);
+        eps = norm->get_eps();
+        epsMode = norm->get_eps_mode() == ngraph::op::EpsMode::MAX ? NormEpsMode::MAX : NormEpsMode::ADD;
         across_spatial = ngraph::shape_size(op->get_input_shape(AXES)) != 1;
+        cornerCase = op->get_input_shape(AXES).empty();
```

Comment:
Could you please add comment what actually cornerCase means. (Can be copied from the spec)

Reply:
added from spec